packages/form-js-editor/src/features/properties-panel/entries/factories/simpleRangeIntegerEntryFactory.js

```diff
@@ -39,31 +40,39 @@ const SimpleRangeIntegerEntry = (props) => {
     path,
     field,
     editField,
-    min,
-    max,
-    defaultValue
+    min = Number.MIN_SAFE_INTEGER,
+    max = Number.MAX_SAFE_INTEGER
   } = props;
 
   const debounce = useService('debounce');
 
   const getValue = () => {
-    const value = get(field, path, defaultValue);
-    return Number.isInteger(value) ? value : defaultValue;
+    const value = get(field, path);
+    const isValid = isValidNumber(value) && Number.isInteger(value);
+    return isValid ? value : null;
   };
 
-  const setValue = (value) => {
-    editField(field, path, value);
+  const setValue = (value, error) => {
+    if (error) {
+      return;
+    }
+
+    editField(field, path, Number(value));
   };
 
-  return NumberFieldEntry({
+  return TextFieldEntry({
     debounce,
     label,
     element: field,
-    step: 1,
-    min,
-    max,
     getValue,
     id,
-    setValue
+    setValue,
+    validate: (value) => {
+      if (value === undefined || value === null || value === '') return;
```

Comment:
If the minimum is 1 we shouldn't allow nil values

Reply:
No we do, because there's always the "not configured" state with our properties, which goes back to a certain default.